### ISSUE TITLE
RHEL6 defines SO_REUSEPORT but it doesn't work

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7159]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)| @tomprince reported|
|-|-|
|Trac ID|trac#7159|
|Type|defect|
|Created|2014-04-15 20:12:31Z|
|Branch|https://github.com/twisted/twisted/tree/rhel6-broken-so-reuseport-7159|

<details><summary>Searchable metadata</summary>

```
trac-id__7159 7159
type__defect defect
reporter__tom_prince tom.prince
priority__normal normal
milestone__ 
branch__branches_rhel6_broken_so_reuseport_7159 branches/rhel6-broken-so-reuseport-7159
branch_author__tomprince tomprince
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1397592751000000 1397592751000000
changetime__1397657811000000 1397657811000000
version__None None
owner__tom_prince tom.prince

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_1"></a>|@tomprince commented|
|-|-|

(In [42285]) Branching to rhel6-broken-so-reuseport-7159.

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_2"></a>|@tomprince commented|
|-|-|

This fixes the persistent error on the rhel builder.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set owner to @tomprince|
|-|-|

Looks good.  The build looks pretty good, there is existing test coverage (hence, presumably, the failure) for this functionality (`test_multiListen`).

Please merge.

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_4"></a>|@tomprince set status to `closed`|
|-|-|

(In [42368]) Merge rhel6-broken-so-reuseport-7159: RHEL6 defines SO_REUSEPORT but it doesn't work.

Author: tom.prince
Reviewer: exarkun
Fixes: [#7159](https://github.com/twisted/twisted/issues/7159)

twisted.internet.interfaces.IReactorMulticast.listenMultiple no longer fails
in that case.
